### DISCUSSION TITLE
Issue creating main.yml in GitHub Actions workflow

### DISCUSSION THREAD

**Discussion by notrue** [community/community#25022] [Actions]
<p>When choosing setup a workflow in GitHub Actions it generates <strong>main.yml</strong> file in a path with repeating name:<br>
<a href="https://github.com/notrue/brickset-scraper/commit/733cd02a95ea2c5a91ea7052c27e1342b9251fe2" rel="nofollow noopener">https://github.com/notrue/brickset-scraper/commit/733cd02a95ea2c5a91ea7052c27e1342b9251fe2</a> </p>
<p>Anyone has this issue?</p>

**Comment by mscoutermarsh** [comment, marked as answer]
<p>We have the wrong link in the UI. It’s getting fixed today.</p>
<p>Thank you!</p>

**Comment by notrue** [comment]
<p>Thank you,</p>
<p>I guess it is not only UI issue, because I pulled to my local and it has this long repeating path as well</p>
